### PR TITLE
Adding instructions on how to update blog

### PR DESCRIPTION
**Pull request by psschwei**
## Proposed Changes 

Adds a README for the blog that details how the blog works, how to make updates, etc.

/assign @omerbensaadon @csantanapr

### CODE REVIEW

File: blog/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+If a major change to the site navigation is made (for example, adding a new section to the top navigation tabs), then the change will need to be made in both `docs/config/nav.yml` and `docs/blog/config/nav.yml`.
+
+For changes that are not top-level (i.e. adding a subsection to the "Getting started" guide or creating a new category of blog post), the change only needs to be made in the relevant section, as it's invisible to the other (for example, the subsectino of the "Getting started" guide only needs to be made in `docs/config/nav.yml`)
+
+## Common files between main site and blog, also known as non nav.yml files
+
+All fiels in `docs/overrides`, `docs/images`, and `docs/stylesheets` are copied to the blog at build-time. 
+
+What this means is that any changes to files in those directories (for example, updating the main site layout editing `docs/stylesheets/extra.css`) will go live _on the blog_ as soon as those changes are pushed to `main`. In contrast, on the main site those changes will appear in the development branch and only go live on what users see when they are cherry-picked into the current release branch.
```
Comment (snneji) [inline blog/README.md:95]:
```suggestion
What this means is that any changes to files in those directories (for example, updating the main site layout editing `docs/stylesheets/extra.css`) will go live _on the blog_ as soon as those changes are pushed to `main`. In contrast, on the main site those changes will appear in the development branch and only go live when they are cherry-picked into the current release branch.
```

File: blog/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Updating the site navigation
+
+If a major change to the site navigation is made (for example, adding a new section to the top navigation tabs), then the change will need to be made in both `docs/config/nav.yml` and `docs/blog/config/nav.yml`.
+
+For changes that are not top-level (i.e. adding a subsection to the "Getting started" guide or creating a new category of blog post), the change only needs to be made in the relevant section, as it's invisible to the other (for example, the subsectino of the "Getting started" guide only needs to be made in `docs/config/nav.yml`)
+
+## Common files between main site and blog, also known as non nav.yml files
+
+All fiels in `docs/overrides`, `docs/images`, and `docs/stylesheets` are copied to the blog at build-time.
```
Comment (snneji) [inline blog/README.md:93]:
```suggestion
All files in `docs/overrides`, `docs/images`, and `docs/stylesheets` are copied to the blog at build-time. 
```

File: blog/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Updating the blog
+
+When a new blog post is created, it will also need to be added to the blog navigation (i.e. to `docs/blog/config/nav.yml`) in the appropriate section. 
+
+## Updating the site navigation
+
+If a major change to the site navigation is made (for example, adding a new section to the top navigation tabs), then the change will need to be made in both `docs/config/nav.yml` and `docs/blog/config/nav.yml`.
+
+For changes that are not top-level (i.e. adding a subsection to the "Getting started" guide or creating a new category of blog post), the change only needs to be made in the relevant section, as it's invisible to the other (for example, the subsectino of the "Getting started" guide only needs to be made in `docs/config/nav.yml`)
```
Comment (snneji) [inline blog/README.md:89]:
```suggestion
For changes that are not top-level (i.e. adding a subsection to the "Getting started" guide or creating a new category of blog post), the change only needs to be made in the relevant section, as it's invisible to the other (for example, the subsection of the "Getting started" guide only needs to be made in `docs/config/nav.yml`)
```

**Review by snneji — CHANGES_REQUESTED** [review]
Found a few typos other than that, looks good to me. @csantanapr or @omerbensaadon would be better placed to comment on the completeness of the information provided.

**Comment by omerbensaadon** [comment]
Going to follow these instructions now to post a new blog

**Comment by abrennan89** [comment]
@omerbensaadon any update on whether this can be merged? Can you do a review please?

**Comment by snneji** [comment]
@omerbensaadon @csantanapr, would you like to review this PR? If not, since this is contributor and not user documentation, I think it's good enough to merge. If anyone has any issues with the procedure, they can submit a followup PR.

**Comment by snneji** [comment]
Gonna merge, any changes necessary can go in a new PR.

/lgtm
/approve

**Review by snneji — APPROVED** [review]

**Pull request merged**